submissions/nataliaArtsiukh/TinyJSWorld/index.js

```diff
@@ -0,0 +1,44 @@
+const man = {
+   hands: 2,
+   legs: 2,
+   name: 'Bob',
+   gender: 'male',
+   saying: 'I\'m vegan',
+   friends: 'no one'
+ };
+ const woman = {
+   hands: 2,
+   legs: 2,
+   name: 'Mary',
+   gender: 'female',
+   saying: 'Hi everyone',
+   friends: 'Tom, Rex, Halle'
+ };
+ const cat = {
+   hands: 0,
+   legs: 4,
+   name: 'Tom',
+   gender: 'male',
+   saying: 'Mrrrr',
+   friends: 'Mary, Halle'
+ };
+ const dog = {
+   hands: 0,
+   legs: 4,
+   name: 'Rex',
+   gender: 'male',
+   saying: 'Woof',
+   friends: 'Mary'
+ };
+ const catwoman = {
+   __proto__:cat,
```

Comment:
When you go through JS OO module (prototypical inheritance) please pay special attention to how correctly bind prototypes.

You do not need to fix anything here at this stage.